linera-execution/src/graphql.rs

```diff
@@ -21,7 +21,18 @@ doc_scalar!(
     Epoch,
     "A number identifying the configuration of the chain (aka the committee)"
 );
-doc_scalar!(Recipient, "The recipient of a transfer");
+doc_scalar!(Recipient, r#"
+The recipient of a transfer.
+
+An example of a `Recipient`:
+```graphql
```

Comment:
Technically this is json. Perhaps the comment could mention GraphQL?

Reply:
I don't think this is JSON. JSON always has string keys.

E.g.:

```
{
    "Account" : {
        "chain_id": <chain_id>
    }
}
```

Reply:
gotcha